files/xorg.conf.nvidia

```diff
@@ -36,6 +36,7 @@ Section "Device"
     Identifier     "Device0"
     Driver         "nvidia"
     VendorName     "NVIDIA Corporation"
+    BusID          "PCI:0:30:0"
```

Comment:
Did you find this change required to make the GPU to work? Making the BusID explicit make us less resilient to changes in the hardware.